corehq/apps/saved_reports/tests/end_to_end/test_scheduled_reports.py

```diff
@@ -0,0 +1,89 @@
+from django.test import TestCase
+from django.core import mail
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.users.models import UserRole, Permissions, WebUser, CouchUser
+from pillowtop.es_utils import initialize_index_and_mapping
+from corehq.util.elastic import ensure_active_es, ensure_index_deleted
+from corehq.pillows.mappings.case_mapping import CASE_INDEX_INFO
+
+from ...models import ReportNotification, ReportConfig
+
+
+class TestScheduledReports(TestCase):
+
+    def test_scheduled_reports_sends_to_recipients(self):
+        saved_report = self.create_saved_report()
+        scheduled_report = self.create_scheduled_report([saved_report], ['test1@dimagi.com', 'test2@dimagi.com'])
+        scheduled_report.send()
+
+        self.assertEqual(len(mail.outbox), 2)
```

Comment:
This assertion is is implied in the one two lines below, correct? That is, there is no way that this would fail if the one below will not also fail? I find the information given by the assertion below (showing which email addresses were missing or unexpected ones that were present) to be much more useful than something like `AssertionError: 3 != 2`. What do you think of removing this assertion?